packages/block-editor/src/components/link-control/search-item.js

```diff
@@ -149,7 +149,15 @@ function getVisualTypeName( suggestion ) {
 		return 'blog home';
```

Comment:
```
const ICONS_MAP = {
	post: postList,
	page,
	post_tag: tag,
	category,
	attachment: file,
};
```

Please could we add `media: media` to this icons map? 🙏 Otherwise Media files have no icon 😞 

You'll need to import the `media` icon first.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/58638/commits/de5d6d104cf75a96645b7dcd97c65d4892899c18